instrumentation/opentelemetry-instrumentation-confluent-kafka/tests/test_instrumentation.py

```diff
@@ -58,3 +60,14 @@ def test_instrument_api(self) -> None:
 
         consumer = instrumentation.uninstrument_consumer(consumer)
         self.assertEqual(consumer.__class__, Consumer)
+
+    def test_context_setter(self) -> None:
+        context_setter = KafkaContextSetter()
+
+        carrier_dict = {"key1": "val1"}
+        context_setter.set(carrier_dict, "key2", "val2")
+        self.assertGreaterEqual(carrier_dict.items(), {"key2": "val2".encode()}.items())
+
+        carrier_list = [("key1", "val1")]
+        context_setter.set(carrier_list, "key2", "val2")
+        self.assertTrue(("key2", "val2".encode()) in carrier_list)
```

Comment:
Same here

Reply:
and yes here the test is ensuring that the new tuple being added on line 72 exists in the carrier_list (if that's your question)